src/views/Migration.vue

```diff
@@ -0,0 +1,356 @@
+				<button type="submit">Submit &rarr;</button>
+			</form>
+		</div>
+
+		<div v-if="step == 2">
+			<h1>Confirm</h1>
+			<label for="c-email">Email address:</label>
+			<input id="c-email" disabled readonly type="email" :value="user.email" placeholder="(no email)">
+			<button @click="confirm">Confirm</button>
+		</div>
+
+		<div v-if="step == 3">
+			<h1>Migration process started</h1>
+			<p>Your account migration has been requested.</p>
+
+			<h1>Next steps</h1>
+			<p>If everything went well, you'll be able to login on Mana Launcher using the provided email and password.</p>
+			<p>If you enabled 2FA, you'll also need the token which has been sent to your email.</p>
+
+			<br><br><br>
+			<h1>Can't find the account you were looking for?</h1>
+			<p>If data inserted here is incorrect or a Vault account already exists bound to that email, the migration will not happen.</p>
+
+			<h1>Still need help?</h1>
+			<p>Feel free to <router-link :to="{ name: 'support' }">contact us</router-link> for further assistance.</p>
+		</div>
+
+		<div class="g-recaptcha" id="recaptcha-container"
+			:data-sitekey="recaptcha_key"
+			data-size="invisible">
+		</div>
+	</main>
+</template>
+
+<script lang="ts">
+import { Options, Vue } from "vue-class-component";
+import reCAPTCHA from "@/reCAPTCHA";
+
+@Options({})
+export default class Recovery extends Vue {
+	step = -3; // ask to use reCAPTCHA
+	nextStep = 1; // first step after reCAPTCHA confirmation
+	user = {
+		email: "",
+		name: "",
+		totp: false,
+		pwd: "",
+	};
+
+	recaptcha_key = process.env.VUE_APP_RECAPTCHA;
+
+	async mounted () {
+		// already loaded (user returned to this page)
+		if (reCAPTCHA.isReady) {
+			if (this.step == -3) {
+				this.step = this.nextStep;
+			}
+
+			await this.$nextTick();
+			reCAPTCHA.instance.render("recaptcha-container", {
+				sitekey: process.env.VUE_APP_RECAPTCHA,
+				size: "invisible",
+			});
+			reCAPTCHA.instance.reset();
+
+			if (this.step == 1) {
+				(this.$refs._email as HTMLInputElement).focus();
+			}
+		}
+	}
+
+	async start () {
+		this.step = -4;
+
+		try {
+			await reCAPTCHA.load();
+			this.step = this.nextStep;
+			await this.$nextTick();
+
+			if (this.step == 1) {
+				(this.$refs._email as HTMLInputElement).focus();
+			}
+		} catch (err) {
+			this.step = -1
+		}
+	}
+
+	async checkEmail () {
+		this.step = reCAPTCHA.isReady ? 2 : -1;
+		// XXX: any actual checks needed here?
+	}
+
+	private sleep (milliseconds: number) {
+		return new Promise(resolve => setTimeout(resolve, milliseconds));
+	}
+
+	async confirm () {
+		reCAPTCHA.instance.execute();
+		let token = "";
+		console.log("Confirmed")
+
+		// the recaptcha API doesn't play nice with Vue
+		while (!(token = reCAPTCHA.instance.getResponse())) {
+			await this.sleep(1000);
+		}
+		console.log("Captcha OK")
+
+		// ${process.env.VUE_APP_API}
+		// https://api.tmw2.org:13370/
+		const req = new Request(`http://localhost:13370/tmwa_auth`, {
```

Comment:
This obviously needs fixing, dummy.
```suggestion
		const req = new Request(`${process.env.VUE_APP_PYAPI}/tmwa_auth`, {
```